### PR TITLE
Document instructions to build/package locally

### LINKED ISSUE #282: Document the way to add/build/install the driver
Flatpak driver update lags quite a bit. As I'm writing this, the newest driver 560.35.03 is added to this repo but is still absent at flathub. Because of that any non-basic GUI software fails to launch completely (Krita, Blender, Shotcut etc.). There should be a simple script that does everything needed to install the driver update locally, without waiting for flathub. I suppose the procedure is trivial, the binary driver is downloaded, unpacked, the required libraries are installed and everything just works. This issue clearly happens *way too often* to ignore and tell people to wait.

I found [these instructions](https://github.com/flathub/org.freedesktop.Platform.GL.nvidia/issues/107#issuecomment-1126341081) which for whatever reason don't install something correctly. First, the `flathub` repo should be added manually for `build.sh` to work at all. I have `flathub` added for `system` but not `user`, everything worked just fine. I added this repo for user, now when I install anything flatpak asks me which repo of the two to use. I assume this can be mitigated somehow.

I did what the instructions say, the binary driver was downloaded but something went wrong (even though there were no errors) and nothing actually changed. `flatpak history` shows `deploy install org.freedesktop.Platform.GL.nvidia-560-35-03`, I'm not sure what else I can check.

Ideally, I see this working like this:
- launch `./install.sh` (or similar)
- it runs `nvidia-smi` to get the current driver version
- does everything needed to download and repack the driver
- installs it locally just like any other flatpak
- all software starts working as before

### PR DESCRIPTION
**Pull request by guihkx**
Fixes #282

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,97 @@
...
+# In this example, only the 560.35.03 driver will be built.
+# You can also build multiple versions by separating them with a space.
+echo 'DRIVER_VERSIONS="560.35.03"' >> versions.sh
+```
+
+5\. Install the appropriate 1.6 Freedesktop Platform/SDK for your CPU architecture:
+
+```bash
+flatpak --user install --no-related flathub "org.freedesktop.Platform/$(flatpak --default-arch)/1.6"
+flatpak --user install --no-related flathub "org.freedesktop.Sdk/$(flatpak --default-arch)/1.6"
```
Comment (TingPing) [inline README.md:42]: I usually prefer recommending they just add `--install-deps-from=flathub` to their f-b args.

**Review by TingPing — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (guihkx) [reply README.md:42]: Yeah, sure, but "unfortunately" that ends up also installing more packages than we actually need, namely ffmpeg and video drivers (when they're not already installed).

That's why I opted to installing the Sdk and Platform packages manually; That way we only install what's *actually* needed...

**Comment by guihkx** [comment]
Thanks for the review!

Once this gets merged, can I also interrupt the build on [buildbot.flathub.org](https://buildbot.flathub.org/)?

Because I honestly don't see the point of triggering a new build given we're only changing the README here...

**Comment by TingPing** [comment]
> Once this gets merged, can I also interrupt the build on [buildbot.flathub.org](https://buildbot.flathub.org/)?

Sure, feel free.

**Pull request merged**